utils/rpms/daos.spec

```diff
@@ -554,6 +562,9 @@ getent passwd daos_agent >/dev/null || useradd -s /sbin/nologin -r -g daos_agent
 # No files in a shim package
 
 %changelog
+* Thu May 11 2023 Lei Huang <lei.huang@intel.com> 2.3.107-3
+- Add libcapstone as a new prerequisite package
```

Comment:
You should also add a comment about the new file being installed in the -client RPM here.

Reply:
Updated. Thank you!